sdk/avs/Azure.ResourceManager.Avs/src/Customized/CommonClusterProperties.cs

```diff
@@ -67,12 +67,16 @@ internal CommonClusterProperties(int? clusterSize, AvsPrivateCloudClusterProvisi
         }
 
         /// <summary> The cluster size. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public int? ClusterSize { get; set; }
         /// <summary> The state of the cluster provisioning. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public AvsPrivateCloudClusterProvisioningState? ProvisioningState { get; }
         /// <summary> The identity. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public int? ClusterId { get; }
         /// <summary> The hosts. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public IList<string> Hosts { get; }
```

Comment:
in order to let the customized code work properly, we need to add a `internal set` to these properties.